.sonata-project.yml

```diff
@@ -24,6 +24,10 @@ labels:
   - name: help wanted
     color: '0e8a16'
 
+files:
+  - doc/CONTRIBUTING.md:CONTRIBUTING.md
+  - tools/editorconfig/.editorconfig:.editorconfig
```

Comment:
Yes maybe. Have to test, didn't decide yet.

We also have more complicated file like `.travis.yml` that will needs a twig template...